src/main/sensors/gyro.c

```diff
@@ -101,15 +101,15 @@ STATIC_FASTRAM filterApplyFnPtr gyroFilterStage2ApplyFn;
 STATIC_FASTRAM void *stage2Filter[XYZ_AXIS_COUNT];
 #endif
 
-PG_REGISTER_WITH_RESET_TEMPLATE(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 3);
+PG_REGISTER_WITH_RESET_TEMPLATE(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 4);
 
 PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_lpf = GYRO_LPF_42HZ,      // 42HZ value is defined for Invensense/TDK gyros
     .gyro_soft_lpf_hz = 60,
     .gyro_align = ALIGN_DEFAULT,
     .gyroMovementCalibrationThreshold = 32,
     .looptime = 1000,
-    .gyroSync = 0,
+    .gyroSync = 1,
```

Comment:
I believe we should make this conditional (`ENABLE_GYRO_SYNC` or something like this). Some targets from Airbot don't have gyro interrupt wired for off-board gyros.

Reply:
@digitalentity this is not needed, there is a routine in config.c that handles that
```
#if !defined(USE_MPU_DATA_READY_SIGNAL)
    gyroConfigMutable()->gyroSync = false;
    systemConfigMutable()->asyncMode = ASYNC_MODE_NONE;
#endif
```